src/components/sidenav/sidenav.js

```diff
@@ -55,6 +55,10 @@ angular
  * // Sync check to whether given sidenav is locked open
  * // If this is true, the sidenav will be open regardless of close()
  * $mdSidenav(componentId).isLockedOpen();
+ * // On close callback to handle close, backdrop click or escape key pressed
```

Comment:
Please note that the callback happens BEFORE the close action occurs.
Should we also support/allow a callback return value to "prevent" the close intention?